packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-pwd-recovery-mfa/main/nodejs/integrationsteps.md

```diff
@@ -1,121 +1,122 @@
 
-<div class="common-image-format">
+<div class="common-image-format bordered-image">
 
-![Displays the Select authenticator page that has a field to select email.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-select-auth-nodejs.png)
+![Screenshot of a page displaying a list of authenticators including the email authenticator](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-pwd-recovery-screenshot-select-auth-nodejs.png)
 
 </div>
 
-### 3: The user selects the email authenticator factor
-
-After the user selects the email authenticator, the next step is to call `idx.recoverPassword` with the authenticator type, as shown in the `authenticator.js` file of the SDK sample application. The method returns a status of `Idx.Status:PENDING`. This status indicates that the Okta platform has emailed the verification code to the user's email address and is now awaiting verification. The response also includes a `nextStep` field that requires an email verification code input parameter.
-
-```JavaScript
-router.post('/select-authenticator', async (req, res, next) => {
-  const { idxMethod } = req.getFlowStates();
-  const { authenticator } = req.body;
-  const authClient = getAuthClient(req);
-  const transaction = await authClient.idx[idxMethod]({ authenticator });
-  handleTransaction({ req, res, next, authClient, transaction });
-});
+### 5. Verify identity with the email authenticator
+
+Next, the user selects and verifies their identity with the email authenticator challenge. The email authenticator supports OTP and magic links, and you can integrate both methods into your application. Learn more about integrating the email authenticator challenge by visiting the [Okta email integration guide](/docs/guides/authenticators-okta-email/nodeexpress/main/#integrate-email-challenge-with-magic-links).
+
+### 6. Display password reset page
+
+After the user verifies their email, `OktaAuth.idx.proceed()` returns a response indicating that user is allowed to reset their password. Specifically, `IdxTransaction` returns a `status` of `PENDING` and `nextStep.name` equal to `reset-authenticator`.
+
+```javascript
+{
+  status: "PENDING",
+  nextStep: {
+    name: "reset-authenticator",
+    inputs: [
+      {
+        name: "password",
+        label: "New password",
+        secret: true,
+        type: "string",
+        required: true,
+      },
+    ],
+  },
+}
 ```
 
-### 4: Create the code verification page and reset the password
-
-After the call to `idx.recoverPassword` has responded with a status of `Idx.Status:PENDING`, the user is redirected to a code verification page. The page must display a field to enter a code and a button or link to send the code to Okta for the email verification.
+Using this response, create a password reset page that allows the user to enter their new password.
```

Comment:
You're not using the response at all to create the page. Just taking the cue that it should be made.

```suggestion
Create a password reset page that allows the user to enter their new password.
```

Reply:
In regards to the sample app, your right. I placed "Using this response" because the response includes content to create the New password label. Let's talk about this one, but I'm okay with removing it.

Reply:
updated.